client/src/components/Message.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import messages from '../../../sampleData';
```

Comment:
does this need file type? i know we can add some thing to the webpack so we dont have to include .js or .jsx in our import statements.